src/rpc/dial.rs

```diff
@@ -753,9 +753,10 @@ async fn maybe_connect_via_webrtc(
         Ok(resp) => resp,
         Err(e) => {
             if e.code() == tonic::Code::Unimplemented {
-                OptionalWebRtcConfigResponse::default();
+                tonic::Response::new(OptionalWebRtcConfigResponse::default())
```

Comment:
(flyby) we weren't correctly using a default response in the `Unimplemented` case before.